src/modules/apis/pointer-capture/index.ts

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+export const hasSupport = (): boolean => {
+  return true;
```

Comment:
@supminn is it supported by all the browsers?

Reply:
I have updated the implementation for `hasSupport`.